src/chttpd/src/chttpd_misc.erl

```diff
@@ -150,6 +160,57 @@ handle_task_status_req(#httpd{method='GET'}=Req) ->
 handle_task_status_req(Req) ->
     send_method_not_allowed(Req, "GET,HEAD").
 
+handle_scheduler_req(#httpd{method='GET', path_parts=[_,<<"jobs">>]}=Req) ->
```

Comment:
We moved it from cloudant's specific handler to chttpd. 

The interesting thing, it doesn't seems like couch_replicator_httpd is used? Maybe I am looking in the wrong place.